btree/btree.c

```diff
@@ -8502,9 +8515,10 @@ static int unmap_node(void* addr, int64_t size)
 static int remap_node(void* addr, int64_t size, struct m0_be_seg *seg)
 {
 	void *p;
+	m0_bcount_t offset = (m0_bcount_t)(addr - seg->bs_addr);
```

Comment:
Use name file_offset instead of offset as it helps while reading the code.

Reply:
Done.